tests/hindsight/jsonschema.4.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
Please add a note to the README (or somewhere) with the provenance of this schema.